code/modules/mob/living/carbon/xenomorph/abilities.dm

```diff
@@ -1371,76 +1362,154 @@ GLOBAL_LIST_INIT(pattern_images_list, list(
 	)
 	use_state_flags = ABILITY_USE_LYING
 	var/datum/buildingpattern/selected_pattern = new /datum/buildingpattern/square2x2
+	/// Holograms are used to show the pattern before placing it
+	var/list/holograms = list()
+	/// timerid before we cleanup the holograms
+	var/cleanup_timer
+	/// how far a hologram can visually move without being remade
+	var/move_range = 4
```

Comment:
confused about the point of this

Reply:
eliminating any major travel time, if you think it's unnecesary I can get rid of it

Reply:
Still unclear to me

Reply:
removed